R/database.R

```diff
@@ -1240,7 +1201,7 @@ get_combined_sv = function(min_vaf = 0,
 #'
 get_manta_sv = function(min_vaf = 0.1,
```

Comment:
Why did you change the default? Do any other functions call this function and, if so, do they assume a certain default?

Reply:
The reason for this is that `get_combined_sv` does not have a FILTER column. So when this parameter is set to TRUE, we would lose all variants that are returned from `get_combined_sv`. This was a workaround, but something I was hoping to discuss here.

Reply:
changing the default just means you can get the results back but it breaks the ability of the user to filter for only the variants that passed manta's filters. What's not clear to me is whether SVAR has already dropped the non-PASS ones from Manta. If it has, then you can assume that NA is equivalent to PASS. Otherwise, we're a bit stuck. Laura should be able to answer this specific question.

Reply:
Yes, I was thinking the same thing. Laura confirmed, the variants we're talking about here are all PASS (i.e NA is equivalent to PASS), so it makes this easier. Let me quickly accommodate for this new information in the code, then we should be good to go.